docs/index.rst

```diff
@@ -113,6 +113,11 @@ To enable this hook, add it as the ``tm.manager_hook`` setting in your app.
        config.include('pyramid_tm')
        # ...
 
+This specific example, using an explicit mode non-threadlocal manager, is
+highly recommended and is shipped as :func:`pyramid_tm.explicit_manager`.
+Simply set ``tm.manager_hook = pyramid_tm.explicit_manager`` in your settings
```

Comment:
I'd need several people to say that is a good idea. I think it's a way more significant / disruptive change than the other changes in 2.0.